tasks/migrations/0001_initial.py

```diff
@@ -0,0 +1,48 @@
+# Generated by Django 3.2.9 on 2021-11-17 20:10
+
+from django.db import migrations, models
+import django.db.models.deletion
+import enumchoicefield.fields
+import tasks.models
+
+
+class Migration(migrations.Migration):
+    initial = True
+
+    dependencies = [
+        ('users', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Task',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('title', models.CharField(max_length=200)),
+                ('created_date', models.DateTimeField(auto_now_add=True)),
+                ('priority', enumchoicefield.fields.EnumChoiceField(default=tasks.models.Priority(1),
```

Comment:
```suggestion
                ('priority', enumchoicefield.fields.EnumChoiceField(default=tasks.models.Priority.MEDIUM,
```

Reply:
It sets to tasks.models.Priority.LOW , is it really matter to change it to tasks.models.Priority.MEDIUM ?

Reply:
I think the main issue is that is less readable with the index instead of the enum.
from my POV if you decided that the default is LOW so its OK to write it as you suggested:
tasks.models.Priority.LOW

Reply:
@aradz1997 , I looked on this PR https://github.com/redhat-beyond/falcon/pull/45 - and they put there MEDIUM,
just don't put a number:) and choose any default you want

Reply:
Pay attention that this file is being generated automatically. The "numbers" are part of the enum library I have added to make the models more readable. I must say that those migrate files shouldn't be such an issue. They are auto generated...